lib/link.js

```diff
@@ -127,6 +127,7 @@ Link.prototype.detach = function() {
 Link.prototype.forceDetach = function() {
   debug('Force-detach for ' + this.name + '. Current state: ' + this.state());
   this.linkSM.forceDetach();
+  this._detached();
```

Comment:
hmm but if we don't emit `Link.Detached` then the session will never cleanup the link. Perhaps we should emit that signal with a `{ force: true }` in the event that the passed in frame is `undefined` or something?